pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line based tool to make presentations from markdown formatted files.
```

Comment:
Minor nitpick: I believe the correct way to phrase this is "markdown-formatted" (with a hyphen).